sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -345,8 +352,30 @@ public void finish(@Nullable SpanStatus status) {
   @Override
   @ApiStatus.Internal
   public void finish(@Nullable SpanStatus status, @Nullable SentryDate finishDate) {
```

Comment:
@adinauer wanna take a look at this, if it has any effect for backend? I don't think it does, but just to make sure we do not introduce some new potential memory leak or something